share/availability/light/options.go

```diff
@@ -0,0 +1,52 @@
+package light
+
+import (
+	"fmt"
+)
+
+// SampleAmount specifies the minimum required amount of samples a light node must perform
+// before declaring that a block is available
+var (
+	DefaultSampleAmount uint = 16
```

Comment:
do we even still need to extract this to a var if it's just the default?

Reply:
Good question, I am keeping it just to avoid refactoring tests that rely on this value for the time being. Such refactor will include usage of the new `WithSampleAmount` option. I am referring to `./nodebuilder/tests/reconstruct_test.go:94`